app/assets/javascripts/components/reward_page_components/EditPersonLink.jsx

```diff
@@ -0,0 +1,33 @@
+class EditPersonLink extends React.Component {
+  constructor(props) {
+    super(props);
+    //this.timer = null;
+    this.timer = null;
+    this.add_emphasis = false;
+  }
+
+  decorate_link(url, text, person) {
+    if (!this.add_emphasis && person === undefined || person.iban !== null) {
+      return (<a href={url}>{text}</a>)
+    }
+
+    this.timer = setInterval(this.performTimerEvent.bind(this), 2000)
+    return (
+      <a href={url} className="tooltipped" data-position="botton" data-tooltip="Vul je bankrekeningnummer in om geld te verdienen!">
+        {text}
+      </a>
+    )
+  }
+
+  performTimerEvent() {
+    $('.tooltipped').tooltip();
+    $('.tooltipped').trigger("mouseenter.tooltip")
```

Comment:
Getest op mobile?

Reply:
ja. is van materialize ook:

<img width="320" alt="file_001" src="https://user-images.githubusercontent.com/617574/41545477-8c339938-731b-11e8-9a6e-01a0905bf70b.png">